credentials/src/test/kotlin/SSITest.kt

```diff
@@ -108,6 +111,40 @@ class SSITest {
     assertTrue(VerifiableCredential.verify(vcJwt, SimpleResolver(didDocument)))
   }
 
+  @Test
+  fun `creates credential status vc with valid vc builder returns vc jwt`() {
```

Comment:
Is credential status intended in this PR?

Reply:
Yea added it for an example that we have the base cred status objects, more robust vc status will come in future prs